### ISSUE TITLE
Closing font file loaded through HexaPDF::FontLoader::FromFile

### ISSUE THREAD

**Issue by harfangk** [gettalong/hexapdf#21]
Hello,

I'd like to first thank you for creating this library. It's incredibly well-documented and very well-written!

As title said, I have a question about closing that font file. In the [docs](https://hexapdf.gettalong.org/api/HexaPDF/FontLoader/FromFile.html) you mentioned that `The file object representing the font file is not closed and if needed must be closed by the caller once the font is not needed anymore.` 

I'm wondering how I can get to that file object in order to close it before it might become a problem.

**Comment by gettalong** [comment]
Tl;dr: `document.fonts.add(...).wrapped_font&.io&.close`

Generally, Ruby automatically closes IO objects once they get garbage collected.

If that is not good enough:

* `HexaPDF::FontLoader::FromFile#call` returns a font wrapper object. There are currently two variants, one for Type1 fonts and one for TrueType fonts.
* Both variants have a `#wrapped_font` attribute that points to "real" font object, i.e. the one that could essentially be used without HexaPDF itself.
* These real font objects are [`HexaPDF::Font::TrueType::Font`](https://hexapdf.gettalong.org/api/HexaPDF/Font/TrueType/Font.html) and [`HexaPDF::Font::Type1::Font`](https://hexapdf.gettalong.org/api/HexaPDF/Font/Type1/Font.html).
* You don't need to bother with the Type1 font objects since they don't need to hold the IO object open.
* That leaves you with the TrueType font object and they have an `#io` attribute which holds the respective IO source (i.e. file object).
